### ISSUE TITLE
Next 12 + postcss 8 + storybook = eldritch occurrences. Carl Pilcher to blame?

### ISSUE THREAD

**Issue by asktree** [storybookjs/storybook#17178]
**Describe the bug**
![image](https://user-images.githubusercontent.com/12001874/148700049-a04b8b40-f8a4-4196-9edc-9c0e0f067102.png)

**To Reproduce**
My package.json: 
```
{
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "storybook": "start-storybook -p 6006",
    "build-storybook": "build-storybook"
  },
  "dependencies": {
    "@headlessui/react": "^1.4.1",
    "@heroicons/react": "^1.0.4",
    "@typeform/embed-react": "^1.1.5",
    "next": "^12.0.7",
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
  },
  "devDependencies": {
    "@babel/core": "^7.16.7",
    "@storybook/addon-actions": "^6.4.9",
    "@storybook/addon-essentials": "^6.4.9",
    "@storybook/addon-links": "^6.4.9",
    "@storybook/react": "^6.4.9",
    "@types/react": "^17.0.26",
    "autoprefixer": "^10.2.6",
    "babel-loader": "^8.2.3",
    "postcss": "^8.3.5",
    "tailwindcss": "^2.2.4",
    "typescript": "^4.4.3"
  }
}
```

Run `npm run storybook`

**System**
Environment Info:

  System:
    OS: macOS 11.3.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
  Binaries:
    Node: 16.13.0 - ~/.volta/tools/image/node/16.13.0/bin/node
    Yarn: 1.22.17 - ~/.volta/tools/image/yarn/1.22.17/bin/yarn
    npm: 8.1.0 - ~/.volta/tools/image/node/16.13.0/bin/npm
  Browsers:
    Chrome: 97.0.4692.71
    Firefox: 95.0.2
    Safari: 14.1
  npmPackages:
    @storybook/addon-actions: ^6.4.9 => 6.4.9 
    @storybook/addon-essentials: ^6.4.9 => 6.4.9 
    @storybook/addon-links: ^6.4.9 => 6.4.9 
    @storybook/react: ^6.4.9 => 6.4.9 
    
    
**Additional context**
When I encountered this bug, VSCode started becoming unresponsive and had to be closed. Also, my dogs started barking, my cat started hissing at the computer, and my kitchen cabinets began flying open and closed violently.

**Comment by nzacca** [comment]
This has something to do with `faker.js` being pulled recently. See here for details: https://github.com/faker-js/faker.js/issues/24

It is not clear to me as yet what the offending package is (storybook doesn't appear to be using faker). Debugging this at the moment.

**Comment by nzacca** [comment]
Issue is with `color.js` which was recently updated here: https://github.com/Marak/colors.js/commits/master

Need to lock down that package to this commit: https://github.com/Marak/colors.js/commit/7ddd6a3d657efb081abb9beddfec26a01a8790a8

**Comment by ljharb** [comment]
I would point to the github tag for v1.4.0 (https://github.com/ljharb/colors.js/commit/baa0e1c7dc50d868354206b9ea71273e3f05f593). Feel free to do so on my fork, as it won't be going away.

**Comment by nzacca** [comment]
Thanks @ljharb .

**Comment by nzacca** [comment]
`cli-table3` has been updated to lock the version. https://github.com/cli-table/cli-table3/pull/251

**Comment by nzacca** [comment]
Suggest closing this in favor of: https://github.com/storybookjs/storybook/issues/17179

**Comment by shilman** [comment]
Closing as dupe to #17179 which has been resolved in 6.4 / 6.5-alpha
